src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -60,8 +72,16 @@ def schedule_auto_resolution():
     silo_mode=SiloMode.REGION,
 )
 @log_error_if_queue_has_items
-def auto_resolve_project_issues(project_id, cutoff=None, chunk_size=1000, **kwargs):
+def auto_resolve_project_issues(
+    project_id,
+    enabled_issue_types: list[Type[grouptype.GroupType]],
```

Comment:
You should provide a default here for the transition, otherwise this can error during deploy if an old version of the code fires this task without this parameter, but a worker running new code picks up the task and expects it.

Reply:
Good catch!